src/Http/Routing/WebRouting.php

```diff
@@ -89,6 +95,15 @@ public function wire(): void
             $this->router->delete("/speakers/{speaker}", "destroy")->name("speakers.destroy");
         });
 
+        $this->router->controller(NewsController::class)->group(function (): void {
+            $this->router->get("/news", "index")->name("news");
+            $this->router->get("/news/create", "create")->name("news.create");
```

Comment:
Please attach middleware auth where needed, or at least to news creation, as you are trying to attach a news to a user, so it would be nice to make sure, that the user is signed in